components/notifications.js

```diff
@@ -57,24 +59,30 @@ function Notification ({ n, fresh }) {
   )
 }
 
-function NotificationLayout ({ children, nid, href, as, fresh }) {
+function NotificationLayout ({ children, n, href, as, fresh }) {
   const router = useRouter()
+  const nidQuery = nid(n)
   if (!href) return <div className={fresh ? styles.fresh : ''}>{children}</div>
   return (
     <div
       className={
-        `clickToContext ${fresh ? styles.fresh : ''} ${router?.query?.nid === nid ? 'outline-it' : ''}`
+        `clickToContext ${fresh ? styles.fresh : ''} ${router?.query?.nid === nidQuery ? 'outline-it' : ''}`
       }
       onClick={async (e) => {
         if (ignoreClick(e)) return
         nid && await router.replace({
           pathname: router.pathname,
           query: {
             ...router.query,
-            nid
+            nidQuery
           }
         }, router.asPath, { ...router.options, shallow: true })
-        router.push(href, as)
+        if (e.ctrlKey) {
```

Comment:
I'd also consider checking `e.metaKey` for mac users using Cmd+Click, though in general I think it's better to handle it via `a` tags instead of event listeners, since `a` tags are accessible. It sounds like you explored this in the PR description but opted against it for now.

Reply:
Yeah, I tried this route but tbh I'm not sure if I'm satisfied by the experience, it works ctrl + click opens in a new tab but it feels lacking like who is going to actually use this. 
Right now this doesn't pass the vibe check.